### ISSUE TITLE
Replace progress bar implementation for file upload

### ISSUE THREAD

**Issue by mcrot** [ContactEngineering/topobank#877]
The current solution for showing a progress bar on file upload is no longer maintained. 
We need a replacement. This is maybe a good solution, just based on vuejs?
https://serversideup.net/file-upload-progress-indicator-with-axios-and-vuejs/

**Comment by mcrot** [comment]
Btw: This error sometimes happens during upload:

```python
Traceback (most recent call last):
  File "/opt/conda/lib/python3.9/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/opt/conda/lib/python3.9/site-packages/django/core/handlers/base.py", line 171, in _get_response
    response = middleware_method(request, callback, callback_args, callback_kwargs)
  File "/opt/conda/lib/python3.9/site-packages/django/middleware/csrf.py", line 297, in process_view
    request_csrf_token = request.POST.get('csrfmiddlewaretoken', '')
  File "/opt/conda/lib/python3.9/site-packages/django/core/handlers/wsgi.py", line 102, in _get_post
    self._load_post_and_files()
  File "/opt/conda/lib/python3.9/site-packages/django/http/request.py", line 362, in _load_post_and_files
    self._post, self._files = self.parse_file_upload(self.META, data)
  File "/opt/conda/lib/python3.9/site-packages/django/http/request.py", line 322, in parse_file_upload
    return parser.parse()
  File "/opt/conda/lib/python3.9/site-packages/django/http/multipartparser.py", line 264, in parse
    chunk = handler.receive_data_chunk(chunk, counters[i])
  File "/opt/conda/lib/python3.9/site-packages/progressbarupload/uploadhandler.py", line 37, in receive_data_chunk
    data['received'] += self.chunk_size

Exception Type: KeyError at /manager/surface/1780/new-topography/
Exception Value: 'received'
```
When looking here at the code of the progress bar handling, it seems that it's buggy and does not handle 
the case when a cache value is not available. Another argument for replacing the library.

**Comment by pastewka** [comment]
We could replace this in conjunction with implementing #173

**Comment by pastewka** [comment]
Resolved with #1012
